libs/auth-api-lib/src/lib/services/delegation-scope.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Logger, LOGGER_PROVIDER } from '@island.is/logging'
+import { Inject, Injectable } from '@nestjs/common'
+import { InjectModel } from '@nestjs/sequelize'
+import { DelegationScopeDTO } from '../entities/dto/delegation-scope.dto'
+import { DelegationScope } from '../entities/models/delegation-scope.model'
+
+@Injectable()
+export class DelegationScopeService {
+  constructor(
+    @InjectModel(DelegationScope)
+    private delegationScopeModel: typeof DelegationScope,
+    @Inject(LOGGER_PROVIDER)
+    private logger: Logger,
+  ) {}
+
+  async create(
+    delegationScope: DelegationScopeDTO,
+  ): Promise<DelegationScope | null> {
+    this.logger.debug('Creating new delegation scope')
+    return await this.delegationScopeModel.create({ ...delegationScope })
```

Comment:
This goes for the whole PR > You can just return the promise: skipping `async` + `await` ... not requesting changes, but it makes the code more readable.